<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 52, 54);
color: rgb(238, 238, 236);</string>
  </property>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>430</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>330</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxHandenessOfTraffic">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>621</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Handedness of traffic</string>
     </property>
     <widget class="QRadioButton" name="radioButtonLeftHandDrive">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Left-hand drive</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonRightHandDrive">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Right-hand drive</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxShowClock">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Show clock</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxAudioAvoidInterference">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>601</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>[Experimental] Avoid taking over the phone's normal audio path</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabVideo">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxResolution">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>621</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>Resolution</string>
     </property>
     <widget class="QRadioButton" name="radioButton480p">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>480p</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton720p">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>720p</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton1080p">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>1080p</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelResolutionWarning">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>561</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;720p and 1080p must be activated in hidden Developer settings of Android Auto application.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="margin">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelResolutionWarningIcon">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>31</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;img src=&quot;:/ico_warning.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxFrameRate">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>621</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Frame rate</string>
     </property>
     <widget class="QRadioButton" name="radioButton60FPS">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>30</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>60 FPS</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton30FPS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>30 FPS</string>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="horizontalSliderScreenDPI">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>531</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>400</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelScreenDPI">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Screen DPI</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelScreenDPIValue">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>380</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>400</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>621</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>OMX Layer index</string>
     </property>
     <widget class="QLabel" name="labelOmxLayerIndex">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Layer index:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBluetoothAddressFormatInfo_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;OMX Layer is used only in case of OMX video output.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBluetoothAddressFormatInfoIcon_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico_info.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxOmxLayerIndex">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabInput">
    <attribute name="title">
     <string>Input</string>
    </attribute>
    <widget class="QCheckBox" name="checkBoxEnableTouchscreen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable touchscreen</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listWidgetButtons">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>621</width>
       <height>381</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelButtons">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Buttons</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPlayButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPauseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxTogglePlayButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Toggle play</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxNextTrackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Next track</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPreviousTrackButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Previous track</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxHomeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Home</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxBackButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxEnterButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>360</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxPhoneButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxCallEndButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Call end</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxVoiceCommandButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>80</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Voice command</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxLeftButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxRightButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxUpButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxDownButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>240</y>
       <width>92</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxScrollWheelButton">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>280</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Scroll wheel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonSelectAll">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Select all</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonClearSelection">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>140</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear selection</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonShowBindings">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>200</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Show bindings</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabBluetooth">
    <attribute name="title">
     <string>Bluetooth</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxBluetoothAdapter">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>621</width>
       <height>441</height>
      </rect>
     </property>
     <property name="title">
      <string>Bluetooth adapter</string>
     </property>
     <widget class="QRadioButton" name="radioButtonDisableBluetooth">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Disable bluetooth</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonUseExternalBluetoothAdapter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Use external adapter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditExternalBluetoothAdapterAddress">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>120</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelBluetoothAdapterAddress">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>67</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBluetoothAddressFormatInfo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-style:italic; color:#babdb6;&quot;&gt;Provide address in format AA:BB:CC:DD:EE:FF.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelBluetoothAddressFormatInfoIcon">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>160</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico_info.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButtonUseLocalBluetoothAdapter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>151</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Use local adapter</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPairWarningIcon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>206</y>
        <width>31</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico_warning.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelPairWarning">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>210</y>
        <width>501</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>In order to use bluetooth features please remember to pair your device.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="labelSettingsIcon">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>80</y>
     <width>131</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/ico_setting.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonResetToDefaults">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>380</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to defaults</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonResetToDefaults</tabstop>
  <tabstop>pushButtonSave</tabstop>
  <tabstop>radioButtonLeftHandDrive</tabstop>
  <tabstop>radioButtonRightHandDrive</tabstop>
  <tabstop>checkBoxShowClock</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>radioButton30FPS</tabstop>
  <tabstop>radioButton60FPS</tabstop>
  <tabstop>radioButton480p</tabstop>
  <tabstop>radioButton720p</tabstop>
  <tabstop>radioButton1080p</tabstop>
  <tabstop>horizontalSliderScreenDPI</tabstop>
  <tabstop>checkBoxEnableTouchscreen</tabstop>
  <tabstop>listWidgetButtons</tabstop>
  <tabstop>checkBoxPlayButton</tabstop>
  <tabstop>checkBoxPauseButton</tabstop>
  <tabstop>checkBoxTogglePlayButton</tabstop>
  <tabstop>checkBoxNextTrackButton</tabstop>
  <tabstop>checkBoxPreviousTrackButton</tabstop>
  <tabstop>checkBoxHomeButton</tabstop>
  <tabstop>checkBoxPhoneButton</tabstop>
  <tabstop>checkBoxCallEndButton</tabstop>
  <tabstop>checkBoxVoiceCommandButton</tabstop>
  <tabstop>checkBoxLeftButton</tabstop>
  <tabstop>checkBoxRightButton</tabstop>
  <tabstop>checkBoxUpButton</tabstop>
  <tabstop>checkBoxDownButton</tabstop>
  <tabstop>checkBoxScrollWheelButton</tabstop>
  <tabstop>checkBoxBackButton</tabstop>
  <tabstop>checkBoxEnterButton</tabstop>
  <tabstop>pushButtonSelectAll</tabstop>
  <tabstop>pushButtonClearSelection</tabstop>
  <tabstop>pushButtonShowBindings</tabstop>
  <tabstop>radioButtonDisableBluetooth</tabstop>
  <tabstop>radioButtonUseLocalBluetoothAdapter</tabstop>
  <tabstop>radioButtonUseExternalBluetoothAdapter</tabstop>
  <tabstop>lineEditExternalBluetoothAdapterAddress</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
